commands/peers/addpeer.go

```diff
@@ -43,11 +50,65 @@ func addPeerHandler(w http.ResponseWriter, r *http.Request) {
 		rest.SendHTTPError(w, http.StatusInternalServerError, *rsp.OpError)
 		return
 	}
+
+	c := context.EtcdClient
+	// Add member to etcd server
+	mAPI := etcdclient.NewMembersAPI(c)
+	member, e := mAPI.Add(etcdcontext.Background(), "http://"+p.Name+":2380")
+	if e != nil {
+		log.WithField("err", e).Error("Failed to add member")
```

Comment:
The error message could be made better here. Failed to add member into etcd cluster would make it more clear.